storage_service/locations/tests/test_models.py

```diff
@@ -1,9 +1,34 @@
+import os
 
 from django.test import TestCase
 
 from locations import models
 
 
+class TestPackage(TestCase):
```

Comment:
I had started separating the tests as well (see test_api, test_duracloud, test_locations).  Move this to test_package.py?  Eventually we can rename test_models to test_lockss, sinces that's what it really is...

:tada: Tests! :tada: